resources/multisite_drupal_standard.make

```diff
@@ -1194,3 +1194,9 @@ projects[nexteuropa_poetry][type] = module
 projects[nexteuropa_poetry][download][type] = git
 projects[nexteuropa_poetry][download][url] = https://github.com/ec-europa/nexteuropa_poetry.git
 projects[nexteuropa_poetry][download][tag] = 0.1.0
+
+projects[nexteuropa_varnish][subdir] = "contrib"
```

Comment:
It should be "custom" and not contrib, no ?

Reply:
That's debatable, nexteuropa_poetry is also in contrib, for a reason I imagine. Also, if I move it into custom it will be picked up by git.